requirements/doc.in

```diff
@@ -1,10 +1,15 @@
-# Requirements for documentation validation
+# Requirements to build the documentation.
```

Comment:
in the guide the file is called `docs` but it is called `doc` in edx-platform and some other repos I picked at random.

Reply:
The Open edX cookiecutter generates a `doc.in` file, so that one is correct, at least in our context. We could open a PR to docs.openedx.org, changing it to the most common one